src/company_cmd.cpp

```diff
@@ -819,6 +825,12 @@ CommandCost CmdCompanyCtrl(DoCommandFlag flags, CompanyCtrlAction cca, CompanyID
 
 			Company *c = DoStartupNewCompany(false);
 
+			/* In network games, we need to try setting the company manager face here to sync it to all clients.
+			 * If a favorite company manager face is selected, choose it. Otherwise, use a random face. */
+			if (_company_manager_face != 0 && Company::IsHumanID(c->index)) {
+				Command<CMD_SET_COMPANY_MANAGER_FACE>::Post(_company_manager_face);
+			}
+
```

Comment:
This needs to go after the `c == nullptr` check, as otherwise you'll do a nullptr dereference when the company can't be created.

This needs to go in the `if (client_id == _network_own_client_id) {` if, such that it only applies to your own company as created by your own network client, rather than attempting to set your face onto companies created by other network clients.